sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs

```diff
@@ -3456,6 +3601,50 @@ private void SetNameFieldsIfNull()
         #endregion CreatePermission
 
         #region CreateDirectory
+        /// <summary>
+        /// The <see cref="CreateDirectory"/> operation creates a new
+        /// directory in this share.
+        ///
+        /// For more information, see
+        /// <see href="https://docs.microsoft.com/rest/api/storageservices/create-directory">
+        /// Create Directory</see>.
+        /// </summary>
+        /// <param name="directoryName">T
```

Comment:
NIT - remove typo of `T at the end of this line

Reply:
Fixed.